src/AppEntry.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { init } from './store';
 import App from './App';
+import logger from 'redux-logger';
 
 import getBaseName from './Utilities/getBaseName';
 
-const Drift = () => (
-    <Provider store={ init().getStore() }>
-        <Router basename={ getBaseName(window.location.pathname) }>
-            <App/>
-        </Router>
-    </Provider>
-);
+const Drift = () => {
+    const [ registry, setRegistry ] = useState();
+    const store = registry?.getStore();
+
+    useEffect(() => {
+        setRegistry(IS_DEV ? init(logger) : init());
+        return () => {
+            setRegistry(undefined);
+        };
+    }, []);
+
+    return (
+
```

Comment:
You might also need a context and context provider for the registry, in case any of the child components do use it, for example the Inventory might call `getRegistry.register` in it's onload, which would then receive the wrong registry/store. 

The Inventory has an example of how to build and use the context/provider